app/uk/gov/hmrc/helptosave/repo/EmailStore.scala

```diff
@@ -52,44 +50,41 @@ trait EmailStore {
 }
 
 @Singleton
-class MongoEmailStore @Inject() (mongo:   ReactiveMongoComponent,
+class MongoEmailStore @Inject() (mongo:   MongoComponent,
                                  crypto:  Crypto,
-                                 metrics: Metrics)(
-    implicit
-    transformer: LogMessageTransformer)
-  extends ReactiveRepository[EmailData, BSONObjectID](
+                                 metrics: Metrics)(implicit ec: ExecutionContext, transformer: LogMessageTransformer)
+  extends PlayMongoRepository[EmailData](
+    mongoComponent = mongo,
     collectionName = "emails",
-    mongo          = mongo.mongoConnector.db,
-    EmailData.emailDataFormat,
-    ReactiveMongoFormats.objectIdFormats)
-  with EmailStore {
-
-  val log: Logger = new Logger(logger)
+    domainFormat   = EmailData.emailDataFormat,
+    indexes        = Seq(IndexModel(ascending("nino"), IndexOptions().name("ninoIndex").unique(false)))
```

Comment:
I think unique is false by default and need n't to be mentioned?

Reply:
resolved